iina/ThumbnailCache.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  ThumbnailCache.swift
+//  iina
+//
+//  Created by lhc on 14/6/2017.
+//  Copyright © 2017 lhc. All rights reserved.
+//
+
+import Cocoa
+
+class ThumbnailCache {
+
+  static private let sizeofDouble = MemoryLayout<Double>.size
+  static private let sizeofInt64 = MemoryLayout<Int64>.size
```

Comment:
When designing a binary format, we always want a fixed file structure with each byte of data predictable, for maximum compatibility.